k8s/base/ingress-controller.yaml

```diff
@@ -0,0 +1,636 @@
+apiVersion: v1
```

Comment:
This file comes from https://kubernetes.github.io/ingress-nginx/deploy/ . I needed to download it and do some tweaks to things that are specific to our setup. Essentially, this file creates an external Google Load Balancer and an `nginx` controller. 

The tweaks:
- [we're opening port 3000](https://github.com/tablelandnetwork/basin-provider/pull/11/files#diff-277d9645bd55e6669440b63397a4efcfa0f469e0d07ec41d9c89e34f1c79b866R332-R335)
- [we're specifying the static IP address where DNS name points to](https://github.com/tablelandnetwork/basin-provider/pull/11/files#diff-277d9645bd55e6669440b63397a4efcfa0f469e0d07ec41d9c89e34f1c79b866R344)
- also a patch to this file is used to build the final ingress